_data/projects/Maven-Using-CMD.yml

```diff
@@ -0,0 +1,12 @@
+name: Maven-Using-CMD
+desc: Creating a Hello World code project in Java using the Apache Maven build tool and automating it using Jenkins.
+site: https://github.com/NishkarshRaj/Maven-Using-CMD
+tags:
+- Maven
+- Java
+- XML
+- Jenkins
+- Automation
+upforgrabs:
+  name: Help Needed 
+  link: https://github.com/NishkarshRaj/Maven-Using-CMD/issues?q=label%3AUp-for-Grabs
```

Comment:
@NishkarshRaj If you are going to use `Up-for-Grabs` label, then please change name to `Up-for-Grabs` and link to https://github.com/NishkarshRaj/Maven-Using-CMD/labels/Up-for-Grabs

Reply:
Sir, I have updated the link as per your suggestion.
Please review.

Reply:
There's no sir :)